DataAccessLayer/Repositories/Implementation/WordTranslationRepository.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using DataAccessLayer.DataBaseModels;
+using DataAccessLayer.Repositories.Interfaces;
+using Microsoft.EntityFrameworkCore;
+
+namespace DataAccessLayer.Repositories.Implementation
+{
+    public class WordTranslationRepository : ICrudRepository<WordTranslation>
+    {
+        private LanguageSkillsDBContext _db;
+        public WordTranslationRepository(LanguageSkillsDBContext context)
+        {
+            this._db = context;
+        }
+
```

Comment:
Please read about "using" construction for DB context and add it to all methods.
Reason - if your DB context is shared between several methods - it may be blocked or some methods will not update DB properly.